src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.6.2'
+__version__ = '1.7.1'
```

Comment:
Why the jump to 1.7.1?

Reply:
I was anticipating merging the my PR with a minor version bump before this one. I anticipated incorrectly :-)